scripts/dev-e2e.sh

```diff
@@ -5,37 +5,28 @@ set -euo pipefail
 scripts="$(cd "$(dirname "$0")"; pwd)"
 source "$scripts"/init.sh
 
-AOHOME="$scripts/../../armada-operator"
 STATUSFILE="$(mktemp)"
 AOREPO='https://github.com/armadaproject/armada-operator.git'
 ARMADACTL_VERSION='0.19.1'
-JOB_DETAILS=1
+JOB_DETAILS="${JOB_DETAILS:-0}"
+
+if [ "${GITHUB_ACTIONS:-false}" == "true" ]; then
+  AOHOME="$scripts/../armada-operator"
+  JOBSET='armada-spark'
+  JOB_DETAILS=1
+else
+  AOHOME="$scripts/../../armada-operator"
+  now=$(date +'%Y%m%d%H%M%S')
+  JOBSET="armada-spark-$now" # interactive users may run this multiple times on same Armada cluster
+  GITHUB_OUTPUT=/dev/stdout
```

Comment:
Not needed as all writes to `GITHUB_OUTPUT` are are wrapped with ` if [ "${GITHUB_ACTIONS:-false}" == "true" ]`.

```suggestion
```

Reply:
Good catch - fixed in commit 551797d.